test/clientTest.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import
+from past.builtins import basestring
+import unittest
+import time
+import os
+import subprocess
+import signal
+import requests
+
+
+class ClientTest(unittest.TestCase):
+    """A set of test cases for the wes-client."""
+    def setUp(self):
+        """Start a (local) wes-service server to make requests against."""
+        self.wes_server_process = subprocess.Popen('python {} --debug'.format(os.path.abspath('wes_service/wes_service_main.py')), shell=True)
+        time.sleep(5)
+
+    def tearDown(self):
+        """Kill the wes-service server."""
+        os.kill(self.wes_server_process.pid, signal.SIGTERM)
+        while get_server_pids():
+            for pid in get_server_pids():
+                try:
+                    os.kill(int(pid), signal.SIGKILL)
+                    time.sleep(3)
+                except OSError as e:
+                    print(e)
+        unittest.TestCase.tearDown(self)
+
+    def test_md5sum_response(self):
+        endpoint = "http://localhost:8080/ga4gh/wes/v1/workflows"
+        descriptor = "https://dockstore.org:8443/api/ga4gh/v2/tools/quay.io%2Fbriandoconnor%2Fdockstore-tool-md5sum/versions/master/plain-CWL/descriptor/%2FDockstore.cwl"
```

Comment:
I think it might be valuable to leave the dockstore test in considering that we want dockstore to be a source and would it satisfy this to add a second test that simply pulled from from local files.  I see your point and feel like a local file test is pretty valuable.  I also think that working with dockstore is going to be a common use-case and it might be valuable to test that as well as a source.